apps/datahub-e2e/src/e2e/header.cy.ts

```diff
@@ -0,0 +1,125 @@
+describe('header', () => {
+  beforeEach(() => cy.visit('/'))
```

Comment:
You may need to introduce mocks & fixtures
eg.
```
    beforeEach(() => {
        cy.intercept('GET', /your_url/, { fixture: response_fixture }).as('your_alias');
    });
```

Reply:
Yes, Olivia and are planning to look into that in the coming days.